src/main/resources/META-INF/rewrite/bouncycastle-jdk15to18.yml

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2025 the original author or authors.
+# <p>
+# Licensed under the Moderne Source Available License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://docs.moderne.io/licensing/moderne-source-available-license
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.BouncyCastleFromJdk15OnToJdk15to18
```

Comment:
I propose we include this by default in our Java 7 migration recipe, same as we already do for version 8+:
https://github.com/openrewrite/rewrite-migrate-java/blob/d04102342d482a126de498c1549c17ab64edb191/src/main/resources/META-INF/rewrite/java-version-8.yml#L29